contracts/core/Controller.sol

```diff
@@ -469,26 +465,6 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
         return operators[_owner][_operator];
     }
 
-    /**
```

Comment:
do we exceed byte size if we keep `getConfiguration()`?

Reply:
We can choose out of the 4 things that were removed:

- C9
- C10
- C11
- `getConfiguration()`

We can probably have 1 or perhaps 2 back

Reply:
lets keep getConfiguration()

Reply:
https://github.com/ribbon-finance/GammaProtocol-OTC/pull/2/commits/fc0a9441795d8988bf7adfad81a07fcdfed27cda